suite-common/token-definitions/src/tokenDefinitionsUtils.ts

```diff
@@ -14,6 +14,9 @@ export const getContractAddressForNetwork = (
     contractAddress: string,
 ) => {
     switch (networkSymbol) {
+        case 'eth':
```

Comment:
note: would be nice to change this method to network type instead of network symbol, that would speed it even more

Reply:
FYI Impact of my change is pretty small, maybe like 3% and it was hardly measurable (it more my obsession than anything)